HotSpot/AppDelegate.m

```diff
@@ -24,20 +24,26 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     
     // Fake a login
     
-    NSString *username = @"user1";
-    NSString *password = @"password";
-    
-    [PFUser logInWithUsernameInBackground:username password:password block:^(PFUser * user, NSError *  error) {
-        if (error != nil) {
-            NSLog(@"User log in failed: %@", error.localizedDescription);
-        } else {
-            // success
-            
-            // DataManager tests
-            [DataManager test];
-        }
-    }];
-    
+    if (PFUser.currentUser) {
+        // don't need to login, go ahead with tests
+        
+        [DataManager test];
+    }
+    else {
+        NSString *username = @"user1";
```

Comment:
I think Aaron merged his login changes to master. You can integrate with login and delete this else branch before merging with master.

Reply:
A note to myself: https://guides.codepath.org/ios/Building-Data-driven-Apps-with-Parse#6-persisting-user-session